File: 백준/Bronze/1000. A＋B/A＋B.kt
fun main() {
    val input = readLine()!!.split(' ').map(String::toInt)
    val A = input[0]
    val B = input[1]

    if (A > 0 && B > 0 && A < 10 && B < 10) {
        val result = A + B
        println(result)
    } else {
        println("0 < A, B < 10 사이의 정수를 입력하세요.")
    }
}

File: 백준/Bronze/1000. A＋B/A＋B.py
# 문제
# 두 정수 A와 B를 입력받은 다음, A+B를 출력하는 프로그램을 작성하시오.

# 입력
# 첫째 줄에 A와 B가 주어진다. (0 < A, B < 10)

# 출력
# 첫째 줄에 A+B를 출력한다.


# 두 정수 A,B를 입력받는다
A, B = map(int,input().split())

# 입력받은 정수가 범위에 맞는지 검증한다  (0 < A, B < 10)
if A > 0 and B < 10:
    result = A + B
    print(result)
else: print("0 < A, B < 10에 맞게 입력하세요.")
